### ISSUE TITLE
Shape problem with ConfusionMeter

### ISSUE THREAD

**Issue by fahadakhan96** [pyaf/DenseNet-MURA-PyTorch#10]
Hi!

I was trying out your code and found a problem with `ConfusionMeter` in the `train_model` function. The following statement:
```python
confusion_matrix[phase].add(preds, labels.data)
```
in the `train_model` function returns `tuple index out of range`.

On exploring, I found that the two inputs `preds` and `labels.data` have different shapes. The preds is in the form `Tensor(x)`, whereas labels.data is in the form `Tensor([x])`. The exact line in the [https://github.com/pytorch/tnt/blob/master/torchnet/meter/confusionmeter.py](`confusionmeter.py`) file that seems to be causing this error is as below:
```python
assert predicted.shape[0] == target.shape[0], 'number of targets and predicted outputs do not match'
```
In our code, `preds` is `predicted` and `preds.shape` returns `torch.Size()` which does not have a 0 index hence the `tuple index out of range`

**Comment by fahadakhan96** [comment]
I fixed it by doing `preds = preds.view(1)` in the training function.
